prototype/wof.js

```diff
@@ -18,6 +20,10 @@ function insertWofRecord( wof, next ){
   // sanity check; because WOF
   if( !isValidWofRecord( id, wof ) ) { return next(); }
 
+  // enforce pelias/blacklist-stream exclusions
+  let peliasGID = util.format('whosonfirst:%s:%d', wof['wof:placetype'], id);
```

Comment:
Personally I've been phasing out using `util.format` for [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). They're way easier to read in my opinion and allow removing a require.